package.json

```diff
@@ -24,10 +24,8 @@
   "homepage": "https://github.com/Financial-Times/S3O-middleware#readme",
   "dependencies": {
     "body-parser": "^1.14.1",
-    "cookie": "^0.2.2",
     "debug": "^2.2.0",
-    "ft-poller": "^2.1.0",
-    "node-rsa": "^0.2.25"
+    "s3o-middleware-utils": "github:financial-times/s3o-middleware-utils"
```

Comment:
@JakeChampion Not yet, have been swamped with a bunch of other projects. Will try to get to that later today!